### PR TITLE
feat: Create BBB recordings overview for admins and enhance admin video area

### LINKED ISSUE #7345: Create BBB recordings overview for admins and enhance admin video area
* Change admin/video-channels. It should be admin/video/
* Create sub slug admin/video/channels and move current video channel info here
* Create sub slug admin/video/recordings and a table with the following overview of videos
   * Thumbnail (linking to video presentation page | Delete
   * Event Name (below icon links to a) Dashboard and b) View linking to public event page)
   * Owner (Show the event owner name and email here)
   * Length
   * Number of Participants
   * Start Time
   * End Time
   * Visibility
   * Video channel (Show the info which video channel the video is using. Right now we only offer BBB)
* When the user accesses admin/video automatically direct the user to admin/video/recordings

The above info is already available on BBB. Compare Greenlight screenshot below.

Screenshot from 2021-05-26 19-49-50

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #7345 
reqd #https://github.com/fossasia/open-event-server/pull/8057

#### Short description of what this resolves:
enable video recordings feature for admin

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by ranjitkshah** [comment]
(recording deletion and role fetch par)t need to discuss in meeting

**Comment by ranjitkshah** [comment]
recording-deletion will be followed up in next PR's

**Review by MeghalBisht — APPROVED** [review]
Works! 
Screenshot from 2021-06-30 11-47-12

File: app/components/forms/admin/video-channel-form.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class VideoChannelForm extends Component<Args> {
     try {
       this.loading = true;
       await this.args.videoChannel.save();
-      this.router.transitionTo('admin.video-channels');
+      this.router.transitionTo('admin.video');
```
Comment (maze-runnar) [inline app/components/forms/admin/video-channel-form.ts:24]: after creating/editing video channel, this should go to `admin/video/channels` but currently going to `admin/video/recordings`

Comment (maze-runnar) [reply app/components/forms/admin/video-channel-form.ts:24]: Also add sorting start/end time.

**Review by maze-runnar — APPROVED** [review]

File: app/templates/admin/video.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="ui grid">
+  {{#if (or (eq this.router.currentRoute.name 'admin.video.channels.index') (eq this.router.currentRoute.name 'admin.video.recordings') )}}
+    <div class="row">
+      <div class="sixteen wide column">
+        <TabbedNavigation @isNonPointing={{true}}>
+          <LinkTo @route="admin.video.recordings" class="item">
+            {{t 'Video Recordings'}}
+          </LinkTo>
+          <LinkTo @route="admin.video.channels.index" class="item">
```
Comment (MeghalBisht) [inline app/templates/admin/video.hbs:9]:
```suggestion
          <LinkTo @route="admin.video.channels" class="item">
```

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**